tensorflow_probability/python/mcmc/transformed_kernel.py

```diff
@@ -254,18 +306,16 @@ def __init__(self, inner_kernel, bijector, name=None):
         inner_kernel=inner_kernel,
         bijector=bijector,
         name=name or 'transformed_kernel')
-    inner_kernel_kwargs = inner_kernel.parameters.copy()
-    target_log_prob_fn = inner_kernel_kwargs['target_log_prob_fn']
+    target_log_prob_fn = get_target_log_prob(self)
```

Comment:
Same effect but probably clearer style to `get_target_log_prob(inner_kernel)` here, rather than `self`.